packages/@react-spectrum/autocomplete/src/SearchAutocomplete.tsx

```diff
@@ -130,8 +132,9 @@ function _SearchAutocompleteBase<T extends object>(props: SpectrumSearchAutocomp
 
   useLayoutEffect(onResize, [scale, onResize]);
 
+  let width = isQuiet ? null : menuWidth;
```

Comment:
Looks like a typescript issue is making the build fail. I think we want to avoid `width` being `null`.  Maybe try:

```suggestion
  let width = isQuiet ? undefined : menuWidth;
```

If that doesn't work, we can look at doing something like:

```jsx
let width;

if (!isQuiet) {
  width = customMenuWidth ? dimensionValue(customMenuWidth) : menuWidth
} 
```

Reply:
Huh that's a little weird, given that both `Picker` and `ComboBox` have the exact same line without producing a lint error. But `null` -> `undefined` in `SearchAutocomplete` does seem to pass `yarn lint` as you suggested.

Sorry I should've run `yarn lint` locally before the CI caught it. I forgot 😅